content/clusters/20H.Beta.V2.md

```diff
@@ -6,11 +6,12 @@ import { LinkExternal } from 'src/components/Link/LinkExternal'
 - <AaMut mut="S:N501"/> has appeared multiple times independently: each can be associated with different accompanying mutations
 - Amino-acid changes are <AaMut mut={'S:N501Y'}/> (nucleotide mutation <NucMut mut={'A23063T'}/>), <AaMut mut={'S:N501T'}/> (nucleotide mutation <NucMut mut={'A23064C'}/>), and <AaMut mut={'S:N501S'}/> (nucleotide mutation <NucMut mut={'A23064G'}/>) -->
 
-This variant is one of 4 "Variants of Concern" (as designated by the WHO):
+This variant is one of 5 "Variants of Concern" (as designated by the WHO):
 - <Var name="20I (Alpha, V1)" prefix=""/> or <Lin name="B.1.1.7" />
 - <Var name="20H (Beta, V2)" prefix=""/> or <Lin name="B.1.351" />
 - <Var name="20J (Gamma, V3)" prefix=""/> or <Lin name="P.1" />
 - <Var name="21A (Delta)" prefix=""/> or <Lin name="B.1.617.2" />
+- <Var name="21K (Omicron)" prefix =""/> or <Lin name="B.1.1.529" />
 
```

Comment:
I noticed that Rajiv had to modify this snippet a document for every WHO variant. It would be great to factor it out into a separate md component and then `import` into each WHO variant document, similarly to how we do this for some of the other content shared across pages, for example here https://github.com/hodcroftlab/covariants/blob/e202e502d825f785ada6bc40ebb404fb3eeb432b/content/clusters/20A.EU2.md?plain=1#L1-L11.

This will help to eliminate duplication, possibility of divergence and to reduce efforts required to add new variants later.

Reply:
I agree Ivan - would be ideal to break this out. I'm afraid I can't make that happen right now, but it would be good to do!!